docs/slotfilling.rst

```diff
@@ -298,45 +264,125 @@ The ``submit()`` method is called when the action is run and all slots are fille
     dispatcher.utter_template('utter_submit', tracker)
     return []
 
+Once this is complete, the form is deactivated.
+
+.. note::
+    Some important things to consider:
 
+    - The slot ``requested_slot`` is automatically added to the domain as an
+      unfeaturized slot. If you want to make it featurized, you need to add it
+      to your domain file.
 
-Important is that a story should only contain the first call to ``FormAction``,
-so a story will look something like this, if all slots are ``unfeaturized``:
+    - Any slots that are already set won't be asked for. E.g. if someone says
+      "I'd like a vegetarian Chinese restaurant for 8 people" the slots ``cuisine``
+      and ``num_people`` won't get asked for.
```

Comment:
`num_people` is tricky here, because in example we use abstract `number`

Reply:
well, i'm not sure what else we could use here

Reply:
maybe we shall add `num_people` as `from_entity` in `formbot` as well?

Reply:
yes, please do!